cli/locking-cli/Cargo.toml

```diff
@@ -1,18 +1,18 @@
 [package]
-name = "single-farming-cli"
+name = "locking-cli"
```

Comment:
This is the file modification from `staking-cli`, but github showing it is modified from `single-farming-cli`